HAB5/hab5.tex

```diff
@@ -250,11 +250,11 @@ \subsection{Milestones}
     \textbf{Task} & \textbf{Complete Date} \\
     \midrule
     Review and Write Up of Current Systems & September 2nd Week \\
-    Mechanical Constraints for Development & September 2nd Week \\
-    Sensor Acquisition, Readout Config & October 2nd Week \\
+    Mechanical Constraints for Development & September 3rd Week \\
+    Acquistion of Data from non-Visual Sensors, Ensuring Output-Processing Compatibility & October 2nd Week \\
```

Comment:
Can you provide some more detail on what "Ensuring Output-Processing Compatibility" means?